### DISCUSSION TITLE
json schema and vscode

### DISCUSSION THREAD

**Discussion by magick93** [cueblox/blox#112] [Ideas]
Hey

I would be cool if schemas automagically generated a schema that was usable in vscode.

See https://dev.to/brpaz/how-to-create-your-own-auto-completion-for-json-and-yaml-files-on-vs-code-with-the-help-of-json-schema-k1i

**Comment by rawkode** [comment]
Hey @magick93,

We are really hoping that the CUE authors handle this via https://github.com/cue-lang/cue/discussions/663

However, I have started experimenting with a language server binary that could make authoring CueBlox configured data pretty awesome.

I'll update this issue as I experiment.
